components/src/dynamo/profiler/spica/tests/test_search_space.py

```
@@ -0,0 +1,383 @@
+
+
+def _config(**ss_overrides) -> SmartSearchConfig:
+    ss = {
+        "model_name": "deepseek-ai/DeepSeek-V3",
+        "hardware_sku": "gb200",
+        "backend": ["trtllm"],
+        "deployment_mode": ["agg"],
+        "gpu_budget": 16,
+    }
+    ss.update(ss_overrides)
+    return SmartSearchConfig(search_space=ss, workload={"trace_path": "/tmp/t.jsonl"})
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/t.jsonl"

(S108)